dev-tools/omdb/Cargo.toml

```diff
@@ -29,7 +29,7 @@ gateway-test-utils.workspace = true
 humantime.workspace = true
 internal-dns.workspace = true
 itertools.workspace = true
-nexus-client.workspace = true
+nexus-client = { workspace = true, features = ["omicron-internal"] }
```

Comment:
Should the feature be moved to the config in the workspace Cargo.toml? That way everything inside omicron will get it but it won't affect anything outside Omicron.

Reply:
Unfortunately, it does seem to effect things outside of omicron.  It is at least behaving that way in my patched-dep test environment.